setup.py

```diff
@@ -310,7 +310,7 @@ def run(self):
         'Topic :: Multimedia :: Video :: Display',
         'Topic :: Multimedia :: Video :: Non-Linear Editor',
         'Topic :: Software Development :: Libraries :: Python Modules',
-        'License :: Other/Proprietary License',
+        'License :: OSI Approved :: Apache Software License',
```

Comment:
If anyone is curious, PyPI doesn't know about Apache 2.0 and this is what's used for Apache 2.0. See https://pypi.org/classifiers/.